cpp/modules/deck.gl/layers/src/line-layer/line-layer.cpp

```diff
@@ -130,7 +129,22 @@ void LineLayer::drawState() {  // {uniforms}
 }
 
 auto LineLayer::_getModel(wgpu::Device device) -> std::shared_ptr<lumagl::Model> {
-  return std::shared_ptr<lumagl::Model>{nullptr};
+  auto positions = std::make_shared<garrow::Field>("positions", wgpu::VertexFormat::Float3);
+  std::vector<std::shared_ptr<garrow::Field>> attributeFields{positions};
+  auto attributeSchema = std::make_shared<lumagl::garrow::Schema>(attributeFields);
+
+  // TODO(ilija@unfolded.ai): **arrow**::Fields are already being specified in initializeState, consolidate?
+  auto sourcePosition = std::make_shared<garrow::Field>("instanceSourcePositions", wgpu::VertexFormat::Float3);
+  auto targetPosition = std::make_shared<garrow::Field>("instanceTargetPositions", wgpu::VertexFormat::Float3);
+  auto color = std::make_shared<garrow::Field>("instanceColors", wgpu::VertexFormat::Float4);
+  auto width = std::make_shared<garrow::Field>("instanceWidths", wgpu::VertexFormat::Float);
+
+  std::vector<std::shared_ptr<garrow::Field>> instancedFields{sourcePosition, targetPosition, color, width};
```

Comment:
We dont need all the temp variables for the fields, they already contain names. Just specify `std::vector<std::shared_ptr<garrow::Field>>{...}` in the make_shared param